request.js

```diff
@@ -1226,7 +1232,7 @@ Request.prototype.onRequestResponse = function (response) {
     }
 
     var responseContent
-    if (self.gzip && !noBody(response.statusCode)) {
+    if ((self.gzip || self.brotli) && !noBody(response.statusCode)) {
```

Comment:
isn't this going to decode `gzip` response even if the `gzip` option is not set?